packages/contact-center/task/src/TaskList/task-list.presentational.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import {TaskListPresentationalProps} from '../task.types';
+
+const TaskListPresentational: React.FunctionComponent<TaskListPresentationalProps> = (props) => {
+  if (props.taskList.length <= 0) {
+    return <></>; // hidden component
+  }
+
+  const {taskList} = props;
+
+  return (
+    <div
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        gap: '10px',
+        padding: '20px',
+      }}
+    >
+      {taskList.map((task, index) => {
+        const callAssociationDetails = task.data.interaction.callAssociatedDetails;
+        const {ani, dn, virtualTeamName} = callAssociationDetails;
+
+        return (
+          <div
+            key={index}
+            style={{
+              display: 'flex',
```

Comment:
same question here.. if we have styles file, why do we need to mention styles here as well?

Reply:
Styles are not working right now, Rajesh is working on fixing the issue. Right now we are maintaining a style object in the file and then using that object, we are not going to use inline styles.